src/components/shared/NavBar/NavBarDropdown.jsx

```diff
@@ -54,6 +54,7 @@ class NavDropdown extends Component {
               <div className="dropdown-content">
                 <Link to="/profile">My Profile</Link>
                 <Link to="/logout">Logout</Link>
+                <Link to="/admin">Admin</Link>
```

Comment:
Nice work here. However, with this implementation, all users can access the admin page. Can we make the access available to only admins?

Reply:
I think this is still a work in progress.